src/components/Dropdown/Dropdown.tsx

```diff
@@ -224,50 +236,91 @@ export default class Dropdown extends AutoControlledComponent<
             getRootProps,
             getToggleButtonProps,
             isOpen,
+            toggleMenu,
             highlightedIndex,
             selectItemAtIndex,
           }) => {
+            const { innerRef, ...accessibilityRootPropsRest } = getRootProps(
+              { refKey: 'innerRef' },
+              { suppressRefError: true },
+            )
             return (
-              <div
-                className={classes.containerDiv}
-                onClick={this.handleContainerClick.bind(this, isOpen)}
-              >
-                {multiple && this.renderSelectedItems(styles)}
-                {search &&
-                  this.renderSearchInput(
-                    getRootProps,
-                    getInputProps,
+              <Ref innerRef={innerRef}>
+                <div
+                  className={classes.container}
+                  onClick={multiple ? this.handleContainerClick.bind(this, isOpen) : undefined}
```

Comment:
Why are we binding this? The `handleContainerClick` is already an arrow function. Am I missing something?